lib/zoom.js

```diff
@@ -25,7 +25,7 @@ export default (container, dimensions, scales, configuration, data, callback) =>
                 const drops = container.selectAll('.drop-line')
                     .selectAll('.drop')
                     .attr('cx',(d,i) => {
-                        return scalingFunction(new Date(d.date))
+                        return scalingFunction(configuration.date(d))
```

Comment:
Okay, that was a good try from me :)
I actually spent a couple hours trying to do that, but since I'm not really a Javascript guy, I was unable call the onZoom function from the test, so I've had to opt for empirical tests.
This is what I've end up with:
```js
    it('should call configuration.date function in onZoom', () => {
        const zoomInit = require('../../lib/zoom').default;
        const data = [ { name: 'foo', data: [{time: 'whatever'}, {time: 'whatever'}] }];
        const configuration = {
            date: () => new Date()
        };

        spyOn(configuration, 'date').and.callThrough();

        // ???

        expect(configuration.date.calls.count()).toEqual(2);
    });
```

I'll try some more later.